src/controller.cpp

```diff
@@ -1317,10 +1317,18 @@ void controller::edit_urls_file() {
 	v->push_empty_formaction();
 	stfl::reset();
 
-	utils::run_interactively(cmdline, "controller::edit_urls_file");
+	int editor_exit_code = utils::run_interactively(cmdline, "controller::edit_urls_file");
+	if (editor_exit_code != 0) {
+		LOG(level::DEBUG, "utils::run_interactively: "
+			"The editor failed with error code %d - command was %s", editor_exit_code, cmdline);
+	}
 
 	v->pop_current_formaction();
 
+	if (editor_exit_code != 0) {
```

Comment:
Not sure if we should check this particular exit code. I notice Vim will return 0 regardless of what I do, be it opening a non-existent file or not having enough permissions to write something.

Can you check how some popular editors behave?